README.md

```diff
@@ -279,9 +279,140 @@ In this case a directory named `4-procs` containing the simulation results outpu
 <!---                             Docker container                                        -->
 <!--- =================================================================================== -->
 
-<h1 id="docker_container">  Docker Container </h1>
-
-
+<h1 id="docker_container">  Docker </h1>
+An alternative way with respect to building from source, is the Docker option. To use this option Docker must be installed first. Please refer to [Docker webpage](https://www.docker.com/products/docker-desktop/) to know more about Docker and how to install it on your machine. The following steps describe how to build a Docker image or pull an existent one from DockerHub, and how to run a Docker container. The last section is a brief guide to perform the same steps but in Singularity, since HPC systems usually use Singularity to handle containers.
```

Comment:
No, that's definitely not an "alternative way," this should be the PREFERRED way!

As such, I would put everything you added right under `Introduction` and before `Building the svFSIplus Program from Source` (painful) so users see this first thing. We could even move everything under `Building the svFSIplus Program from Source` that's related to externals into a separate developer guide. We don't want anybody who doesn't absolutely have to build the externals from scratch anymore.